.gitignore

```diff
@@ -45,3 +45,6 @@ env/
 
 apps/omg_watcher/priv/static/swagger.json
 apps/omg_rpc/priv/static/swagger.json
+
+# Compiled contracts
```

Comment:
obsolete, contracts build to `_build/contracts`. The files under `contracts` should be deleted altogether